daemon.log

```diff
@@ -0,0 +1,862 @@
+Connecting to broker ...
```

Comment:
Why is this file needed?

Reply:
Sorry. I think this got added by mistake.